app/models/concerns/reactable.rb

```diff
@@ -12,4 +12,11 @@ def sync_reactions_count
   def reaction_categories
     reactions.distinct(:category).pluck(:category)
   end
+
+  def public_reaction_categories
```

Comment:
Out of curiosity: what is the difference between `reaction_categories` and `public_reaction_categories`

Reply:
Good question! These are confusing, and `reaction_categories` is no longer used by any non-test callers. (It was previously a list strings which were unique categories, it was used in a loop to only show icons in the stack that had been reacted, but it's now only being used by `public_reaction_categories` and it can be happily inlined there.)

Reply:
Thanks.